engines/scumm/charset-fontdata.cpp

```diff
@@ -628,6 +689,13 @@ CharsetRendererV2::CharsetRendererV2(ScummEngine *vm, Common::Language language)
 			_fontPtr = russianCharsetDataV2;
 		}
 		break;
+	case Common::HE_ISR:
+		if (((_vm->_game.id == GID_MANIAC) || (_vm->_game.id == GID_ZAK)) && (_vm->_game.version == 2)) {
```

Comment:
```suggestion
		if ((_vm->_game.id == GID_MANIAC || _vm->_game.id == GID_ZAK) && _vm->_game.version == 2) {
```
Makes the code easier to read